pkg/api-server/definitions/traffic_log.go

```diff
@@ -5,13 +5,13 @@ import (
 	"github.com/Kong/kuma/pkg/core/resources/model"
 )
 
-var TrafficLoggingWsDefinition = ResourceWsDefinition{
+var TrafficLogWsDefinition = ResourceWsDefinition{
 	Name: "Traffic Logging",
-	Path: "traffic-logging",
+	Path: "traffic-log",
```

Comment:
I think, it should be `traffic-logs`

Reply:
Renamed traffic-log -> traffic-logs as well as traffic-permission -> traffic-permissions